dev/com.ibm.ws.logging.osgi/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -14,76 +14,73 @@
+message.file.name.desc=Name of the file to which message output will be written relative to the configured log directory. The default value is messages.log. This file always exists, and contains INFO and other (AUDIT, WARNING, ERROR, FAILURE) messages in addition to System.out and System.err. This log also contains time stamps and the issuing thread ID. If the log file is rolled over, the names of earlier log files have the format messages_timestamp.log. Avoid trouble: Use the com.ibm.ws.logging.message.file.name property, except in cases where you are trying to change the configuration dynamically after startup.
 
 trace.file.name=Trace file name
-trace.file.name.desc=Name of the file to which trace output will be written relative to the configured log directory. The default value is trace.log.
+trace.file.name.desc=Name of the file to which trace output will be written relative to the configured log directory. The default value is trace.log. The trace.log file is only created if a traceSpecification is set including log levels below INFO. stdout is recognized as a special value, and causes trace to be directed to the original standard out stream. Avoid trouble: Use the com.ibm.ws.logging.trace.file.name property, except in cases where you are trying to change the configuration dynamically after startup.
 #
 # Console log level
 #
 console.level=Console log level
-console.level.desc=The logging level used to filter messages written to system streams. The default value is audit.
+console.level.desc=The logging level used to filter messages written to system streams. The valid values are INFO, AUDIT, WARNING, ERROR, and OFF. By default, the consoleLogLevel is set to the WLP_LOGGING_CONSOLE_LOGLEVEL environment variable (if set) or AUDIT. Note: Before you change this value, consider the information in section "Unable to interact with the Liberty server after modifying the console log level settings" in the topic Developer Tools known restrictions. Avoid trouble: Use the WLP_LOGGING_CONSOLE_LOGLEVEL environment variable or com.ibm.ws.logging.console.level property, except in cases where you are trying to change the configuration dynamically after startup.
 console.level.info=Info, audit, and warning messages will be written to the system output stream. Error messages will be written to the system error stream.
 console.level.audit=Audit and warning messages will be written to the system output stream. Error messages will be written to the system error stream.
 console.level.warning=Warning messages will be written to the system output stream. Error messages will be written to the system error stream.
 console.level.error=Error messages will be written to the system error stream.
-console.level.off=No server output will be written to system streams.  Only JVM output will be written to system streams.
+console.level.off=No server output will be written to system streams. Only JVM output will be written to system streams.
```

Comment:
Change
"No server output will be written to system streams. Only JVM output will be written to system streams."
to
"No server output is written to system streams. Only JVM output is written to system streams."